packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -250,6 +251,18 @@ export type FilterMap = Map<
     filterList: FilterData[];
   }
 >;
+
+export interface ContextMenuData {
```

Comment:
Maybe something like `IrisGridContextMenuData` so it's more specific?

Reply:
Yea, by virtue of being in the `@deephaven/iris-grid` package, it's already specific to IrisGrid... the `IrisGrid` prefix for all the mouse handlers was essentially redundant and unnecessary.
I'll change this though to be consistent with the redundancy we already have.